cmd/katapult/config_test.go

```diff
@@ -41,6 +36,12 @@ func TestConfig(t *testing.T) {
 			apiKey: "",
 			apiURL: "test",
 		},
+		{
+			name:   "test JSON output",
+			apiKey: "test",
+			apiURL: "test",
+			output: "json",
+		},
```

Comment:
We should probably test YAML output as well as JSON in all commands that supports it.

And for the config command here specifically, I think we should probably use different values for each option to guarantee that the values aren't mixed up. Something like this:

```suggestion
		{
			name:   "json output",
			apiKey: "testKey",
			apiURL: "testURL",
			output: "json",
		},
		{
			name:   "yaml output",
			apiKey: "testKey",
			apiURL: "testURL",
			output: "yaml",
		},
```